app/tui/help.py

```diff
@@ -69,4 +66,4 @@ def _get_key_combo_text(self, key_combos: List[str], label: str):
         return HTML(f'<b>&lt;{first}&gt;</b> {label}')
 
     def __pt_container__(self):
-        return self.container
+        return self._container
```

Comment:
one thing ive been thinking about, probably makes sense to do after this refactor effort.
But i think we can bolster the info collected during the register_configured_keybinding to include a help text/pretty label and current FocusWidget for the binding. 
Then using the collection of FocusWidget -> keybindings with help to drive what should be displayed in this component dynamically. I think will remove the need for this conditional logic, while also guaranteeing if we add new FocusWidgets in the future - they will have correct keybinding help sections

Reply:
I think this is a great idea, I'm having a hard time wrapping my head around the details of this would work, I'll leave that to you to figure out, 😆